task-def-hub.json

```diff
@@ -57,6 +57,10 @@
         "name": "HUB_MODE",
         "value": "true"
       },
+      {
+        "name": "LOGGING_LEVEL",
+        "value": "${logging_level}"
```

Comment:
I'm curious if you can use `upper` here to make it case insensitive. I'd suspect not as it's a json file.

Reply:
Good idea. It could be used in the tf file.